internal/projection/process/event_enricher.go

```diff
@@ -95,7 +95,11 @@ func GetBaseEnrichedEvent(event *types.Event, cluster map[string]interface{}, ho
 	}
 
 	enrichedEvent := &types.EnrichedEvent{}
-	enrichedEvent.Event.Properties = getProps(event.Payload)
+	props, err := getProps(event.Payload)
+	if err != nil {
+		return nil, err
```

Comment:
Not sure we want to return here. If there is an error with formatting of properties we still want to carry on with the rest of processing. Maybe a warning instead?